### ISSUE TITLE
`data-toggle-is-active` does not update `aria-hidden` attribute

### ISSUE THREAD

**Issue by henrijs** [Twikito/easy-toggle-state#125]
**Describe the bug**
When initializing toggle with `data-toggle-is-active` targets `aria-hidden` attribute is not changed. This happens when `data-toggle-target` is used to identify target, if `data-toggle-target-parent` is used, the bug is not present.

**To Reproduce**
1. Have exact markup like in [Collapsible panel with ARIA](https://twikito.github.io/easy-toggle-state/#collapsible-panel-with-aria).
2. Add `data-toggle-is-active` to button.
3. Reload page and note panel is expanded (working, `is-active` class added to button, panels `aria-hidden` set to `false`)
4. Change attribute from `data-toggle-target-parent` to `data-toggle-target`
5. Reload page and note panel is NOT expanded (does not work, `is-active` class added to button, but panels `aria-hidden` IS NOT set to `true` (value not changed))

**Expected behavior**
`aria-hidden` value changed to `true`

**Screenshots**
None.

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser Firefox
 - Version 122

**Additional context**
None.

**Comment by henrijs** [comment]
I guess that's the way it is described in docs as well.
